app/models/saved_claim.rb

```diff
@@ -148,6 +149,23 @@ def va_notification?(email_template_id)
 
   private
 
+  def validate_schema(schema)
+    JSON::Validator.fully_validate_schema(schema, { errors_as_objects: true })
+  rescue => e
+    Rails.logger.error('Error during schema validation!', { error: e.message, backtrace: e.backtrace, schema: })
+    raise
+  end
+
+  def validate_form(schema, clear_cache)
+    JSON::Validator.fully_validate(schema, parsed_form, { errors_as_objects: true, clear_cache: })
+  rescue => e
+    PersonalInformationLog.create(data: { schema:, parsed_form:, params: { errors_as_objects: true, clear_cache: } },
```

Comment:
Logging the form data here so we can better understand why this method is intermittently failing. This will be removed when we find the issue.